core/storage/feedback/gae_models.py

```diff
@@ -113,14 +113,30 @@ def get_by_exp_and_thread_id(cls, exploration_id, thread_id):
 
     @classmethod
     def get_threads(cls, exploration_id):
-        """Returns an array of threads associated to the exploration.
+        """Returns threads ordered by last updated associated to the
+        exploration.
 
         Does not include the deleted entries.
         """
         return cls.get_all().filter(
             cls.exploration_id == exploration_id).order(
                 cls.last_updated).fetch(feconf.DEFAULT_QUERY_LIMIT)
 
+    @classmethod
+    def get_threads_multi(cls, exploration_ids):
+        """Returns an array of threads ordered by last updated associated to
```

Comment:
Returns an array of threads that ... hang on, do we actually still need this method?